.github/workflows/ci_cpu.yml

```diff
@@ -36,8 +36,12 @@ jobs:
         run: |
           python ts_scripts/install_dependencies.py --environment=dev
       - name: Torchserve Sanity
-        run: |
-          python torchserve_sanity.py
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 60
```

Comment:
Does the timeout apply per run or for all runs?

Reply:
Its 60 minutes per run